servers/services/src/main/java/org/projectnessie/services/impl/NamespaceApiImpl.java

```diff
@@ -289,15 +303,16 @@ private Namespace namespaceFromType(KeyEntry<Content.Type> withType) {
     return Namespace.of(elements);
   }
 
-  private Optional<Content> getExplicitlyCreatedNamespace(Namespace namespace, Ref ref)
-      throws ReferenceNotFoundException {
-    return Optional.ofNullable(getStore().getValue(ref, Key.of(namespace.getElements())));
+  private Optional<Content> getExplicitlyCreatedNamespace(
+      Namespace namespace, WithHash<NamedRef> refWithHash) throws ReferenceNotFoundException {
```

Comment:
Here and below: The function only uses the `Hash` - maybe better to refactor to `Hash hash` (the name's irrelevant)